nemo_curator/datasets/doc_dataset.py

```diff
@@ -43,7 +43,8 @@ def read_json(
         cls,
         input_files: Union[str, List[str]],
         backend: str = "pandas",
-        files_per_partition: int = 1,
+        files_per_partition: Optional[int] = None,
+        blocksize: Optional[str] = "1gb",
```

Comment:
As you mentioned it might be nice to add some of the information from https://github.com/rapidsai/cudf/blob/81cd4a00f8ae0dcde359ac11b53a9b3d855e50e2/docs/dask_cudf/source/best_practices.rst#use-parquet to NeMo Curator's best practices.

Also might be of interest for you: https://github.com/rapidsai/cudf/pull/17250